presto-hive/src/main/java/io/prestosql/plugin/hive/HivePartitionManager.java

```diff
@@ -34,7 +34,6 @@
 import io.prestosql.spi.predicate.Domain;
```

Comment:
commit message is too long. 

> Changed Domain#simplify to avoid constructing new objects to get size of Ranges or DiscreteValues

How do you avoid creating new objects?

Reply:
I was referring to `new ArrayList<>(lowIndexedRanges.values());` in `SortedRangeSet#getOrderedRanges` and similar copying of internal structures in `EquatableValueSet#getValues` just to get their size. Now that's replaced by `getRangeCount` and `getValuesCount`.
I'll shorten the commit message